Formula/pypy3.7.rb

```diff
@@ -115,7 +93,7 @@ def install
   def post_install
     # Precompile cffi extensions in lib_pypy
     # list from create_cffi_import_libraries in pypy/tool/release/package.py
-    %w[_sqlite3 _curses syslog gdbm _tkinter].each do |module_name|
+    %w[_sqlite3 _curses syslog gdbm].each do |module_name|
       quiet_system bin/"pypy3", "-c", "import #{module_name}"
     end
```

Comment:
This whole post_install stanza seems redundant to me. The package.py script already does the cffi build. What happens if you remove it (from all the scripts?)